src/main/java/com/example/Customer.java

```diff
@@ -0,0 +1,9 @@
+package com.example;
+
+public class Customer {
+    private String LoyaltyCard;
```

Comment:
Please review the standard style conventions for Java. Field names start from lowercase letter.